RPerformanceTracking/Private/_RPTClassManipulator.m

```diff
@@ -220,6 +144,8 @@ + (void)_removeSwizzleSelector:(SEL)sel onClass:(Class)recipient types:(const ch
 
 + (_Nullable IMP)implementationForOriginalSelector:(SEL)selector class:(Class)classObj
 {
+    classObj = [self furthestAncestorOfRecipient:classObj
```

Comment:
Local variable instead of modify input parameter?

Reply:
Sure. I've changed it.